### PR TITLE
TC 06.01 | OrganizationFolderTest > View/Configure >Credentials

### PR DESCRIPTION
**Pull request by arturpuchkof**
https://trello.com/c/90EXeuh8/2248-tc-0601-organizationfoldertest-view-configure-credentials

### CODE REVIEW

**Comment by arturpuchkof** [comment]
[TC 06.01 | OrganizationFolderTest > View/Configure >Credentials](https://trello.com/c/90EXeuh8/2248-tc-0601-organizationfoldertest-view-configure-credentials)

**Review by morozanna503 — APPROVED** [review]

**Review by HelenMyronyuk — APPROVED** [review]

File: src/test/java/school/redrover/model/CredentialsPage.java [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 import school.redrover.model.base.BaseMainHeaderPage;
 
 public class CredentialsPage extends BaseMainHeaderPage<CredentialsPage> {
 
+    @FindBy(xpath = "//h1")
+    private WebElement pageHeader;
+
+    public String getTitleText() {
+        return getWait2().until(ExpectedConditions.visibilityOf(pageHeader)).getText();
+    }
```
Comment (nikabenz) [inline src/test/java/school/redrover/model/CredentialsPage.java:16]: Методы размещаем после конструктора

Comment (arturpuchkof) [reply src/test/java/school/redrover/model/CredentialsPage.java:16]: пофиксил

Comment (arturpuchkof) [reply src/test/java/school/redrover/model/CredentialsPage.java:16]: https://github.com/RedRoverSchool/JenkinsQA_06/pull/2797

**Pull request merged**